internal/cmd/backup.go

```diff
@@ -118,6 +120,29 @@ func runBackup(c *cli.Context) error {
 		}
 	}
 
+	// Repositories exclude mirror repositories
+	if !c.Bool("exclude-mirror-repos") && !c.Bool("database-only") {
+		reposFile := filepath.Join(rootDir, "non-mirror-repositories.zip")
+		f, err := os.Create(reposFile)
+		if err != nil {
+			log.Fatal("Create JSON file: %v", err)
+		}
+		repos, err := db.GetRepositoriesExcludeMirror()
+		if err != nil {
+			_ = f.Close()
+			log.Fatal("Non-mirror repositories failed to load %v", err)
+		}
```

Comment:
Hmm, we don't want to backup a list of DB records here, instead we want to backup the actual Git content. Thus we need to loop over the list of repositories returned, and add each of them to ZIP archive.

PS: All repositories are stored in the `conf.Repository.Root` path.